src/DigitalSubscriberAppBanner.tsx

```diff
@@ -0,0 +1,332 @@
+
+    ${from.desktop} {
+        font-size: 20px;
+        margin: ${space[3]}px 0 ${space[4]}px;
+        max-width: 42rem;
+    }
+
+    ${from.leftCol} {
+        max-width: 37rem;
+    }
+
+    ${from.wide} {
+        max-width: 42rem;
+    }
+`;
+
+export const strongSmallSpacer = css`
+    font-weight: 700;
+    margin-top: ${space[5]}px;
+    margin-right: ${space[3]}px;
+    display: inline-block;
+`
+
+export const smallRightSpacer = css`
+    margin-right: ${space[3]}px;
+`
+
+export const primaryButton = css `
+    margin-right: ${space[3]}px;
+    background-color: ${palette.neutral[20]};
+    color: ${palette.neutral[100]};
+    &:hover {
+        background-color: #111;
+    }
+`
+
+export const secondaryButton = css `
+    color: ${palette.neutral[20]};
+`
+
+export const packShot = css`
+    max-width: 100%;
+    max-height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    margin-top: -20px;
+    img {
+        max-width: 100%;
+        max-height: 100%;
+        object-fit: contain;
+    }
+    ${until.tablet} {
+        display: none;
+    }
+`;
+
+export const iconPanel = css`
+    ${from.desktop} {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        align-items: flex-end;
+        height: 100%;
+        padding: ${space[4]}px 0;
+        margin-left: ${space[4]}px;
+    }
+`;
+
+
+export const closeButton = css`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    border: 1px solid ${palette.text.primary};
+    border-radius: 50%;
+    outline: none;
+    background: transparent;
+    cursor: pointer;
+    width: 35px;
+    height: 35px;
+    svg {
+        width: 25px;
+        height: 25px;
+        fill: ${palette.text.primary};
+        transition: background-color 0.5s ease;
+        border-radius: 50%;
+    }
+    :hover {
+        cursor: pointer;
+        background-color: rgba(237, 237, 237, 0.5);
+    }
+    
+    position: absolute;
+    top: 10px;
+    right: 10px;
+`;
+
+export const smallInfoIcon = css`
+    svg {
+        width: 40px;
+        height: 40px;
+        vertical-align: -30%;
+        fill: ${palette.neutral[86]};
+    }
+    left: -60px;
+    display: none;
+    ${until.tablet}{
+        display: block;
+        margin-left: -4px;
+        margin-top: -4px;
+        margin-bottom: 4px;
+    }
+
+`;
+
+export const infoIcon = css`
+    svg {
+        width: 60px;
+        height: 60px;
+        fill: ${palette.neutral[86]};
+    }
+    position: absolute;
+    width: 60px;
+    height: 60px;
+    left: -60px;
+    top: 10px;
+    display: none;
+    @media (min-width: 1430px){
+        display: block;
+    }
+`;
+
+export const storeIcon = css`
+    height: 10px;
+    svg {
+        height: 30px;
+        width: auto;
+        margin-top: ${space[2]}px;
+        padding-right: ${space[2]}px;
+    }
+    display: inline-block;
+    vertical-align: -37.5%;
+    
```

Comment:
Couple of empty lines in funny places like this. Again, I'm not fussed in the slightest though :-)

Reply:
They have now been ousted.